.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,18 +3,7 @@
 1. If this is your first time, please read our contributor guidelines here:
 https://github.com/replicatedhq/kURL/blob/main/CONTRIBUTING.md.
 2. If the PR is unfinished, please mark it as a draft.
--->
```

Comment:
Collateral damage/rebase required?

Reply:
No this looks like a good change
Not sure it should be integrated into this pr, but I'm happy to see the change itself